manifests/sssd/client.pp

```diff
@@ -73,21 +71,6 @@
 
   include 'sssd'
 
-  if $local_domain {
-    $_local_domain_defaults = {
-      'description' => 'LOCAL Users Domain',
-      'min_id'      => $min_id
-    }
-
-    sssd::domain { 'LOCAL':
```

Comment:
Do we want to do a lookup and tell people how to update their Hiera if it's wrong?

Something like:

```
if 'LOCAL' in lookup('sssd::domains', undef, undef, []) {
  # ADD WARNING HERE
}
```

Maybe add a way to silence the message and tell them about it?

Reply:
Trevor I added this.  Please review again.